requirements/pip_tools.txt

```diff
@@ -5,5 +5,8 @@
 #    make upgrade
 #
 click==7.1.2              # via pip-tools
-pip-tools==4.5.1          # via -r requirements/pip_tools.in
```

Comment:
there is a major upgrade here.. will be good to ensure everything still works, not sure what exaclt that means but maybe running some make targets, did we try make test etc?

Reply:
I'll try make test. I upgraded it because I couldn't run `make upgrade` without upgrading it. 😝